src/qibocal/auto/operation.py

```diff
@@ -19,6 +19,8 @@
 """Name of the file where data acquired (arrays) by calibration are dumped."""
 JSONFILE = "conf.json"
 """Name of the file where data acquired (global configuration) by calibration are dumped."""
+RESULTFILE = "result.json"
```

Comment:
Just for consistency:
```suggestion
RESULTSFILE = "results.json"
```